android/src/main/java/com/iterable/reactnative/RNIterableAPIModule.java

```diff
@@ -204,6 +204,11 @@ public void execute(@Nullable Uri url) {
         });
     }
 
+    @ReactMethod
+    public void setReadForMessage(String messageId, boolean read) {
+        IterableApi.getInstance().getInAppManager().setRead(RNIterableInternal.getMessageById(messageId), read);
+    }
```

Comment:
Oh are we supporting mark as read publicly?

Reply:
Sort of, this is a bug fix, you can see it here: https://github.com/Iterable/react-native-sdk/pull/114